tests/test_router.py

```diff
@@ -441,3 +445,31 @@ def test_lookup_fallback_if_migration_directory_not_the_same_as_the_model(self):
             can_migrate_default=True,
             can_migrate_shard=False,
         )
+
+
+class RouterForPostgresIDFieldTest(TransactionTestCase):
+
+    def setUp(self):
+        from django.contrib.auth import get_user_model
```

Comment:
Forgot to remove it lol. I was originally using the main user model but it was setup to use the default shard group, so I created a new one and left that there on accident. Will remove it tomorrow.